lib/queryBuilder/graphUpserter/UpsertNode.js

```diff
@@ -154,6 +128,12 @@ function hasOption(node, optName) {
   }
 }
 
+function decideType(node, defaultType, option, optionType) {
+  return hasOption(node, option)
+    ? optionType || UpsertNodeType.None
+    : defaultType;
+}
+
```

Comment:
@koskimas currently, `optionType` is never provided so this could be simplified. But I was going to use this also to support the `update: true` option...